include/rangers-api/Application/MyApplication.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+namespace app {
+    class MyApplication : public hh::game::GameApplication {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.